### PR TITLE
update slack intergration

### PR DESCRIPTION
**Pull request by the456gamer**
add afk listener
add more events
add command: slack bot needs to be granted the /minecraft command

current features: /minecraft players

update afk msg

### CODE REVIEW

**Comment by the456gamer** [comment]
supersedes #154

File: src/main/java/com/hackclub/hccore/slack/SlackBot.java [hunk 1]
```diff
@@ -51,81 +69,125 @@ public SlackBot(HCCorePlugin plugin) throws Exception {
...
+    app.event(MessageChannelJoinEvent.class, (payload, ctx) -> ctx.ack());
+
+    CommandDispatcher<SlashCommandRequest> dispatcher = new CommandDispatcher<>();
+    // TODO change to get base command from config.yml
+    dispatcher.register(
+        LiteralArgumentBuilder.<SlashCommandRequest>literal("/%s".formatted(commandBase))
+            .then(
+                LiteralArgumentBuilder.<SlashCommandRequest>literal("players").executes(context -> {
+                  StringBuilder message = new StringBuilder(
+                      "Players online (%d/%d)\n\n".formatted(
```
Comment (polypixeldev) [inline src/main/java/com/hackclub/hccore/slack/SlackBot.java:105]: I think that there should be a case added to this where if there's nobody online just say "There are no players online".

**Review by polypixeldev — CHANGES_REQUESTED** [review]

File: src/main/java/com/hackclub/hccore/slack/SlackBot.java (hunk 1 above)
Comment (polypixeldev) [reply src/main/java/com/hackclub/hccore/slack/SlackBot.java:105]: I also think that the "Players online" line of the message should be bolded

Comment (the456gamer) [reply src/main/java/com/hackclub/hccore/slack/SlackBot.java]: done

**Review by polypixeldev — APPROVED** [review]

**Pull request merged**